packages/wallet-sdk/src/connection/RxWebSocket.test.ts

```diff
@@ -1,14 +1,13 @@
 import WS from 'jest-websocket-mock';
-import { Observable } from 'rxjs';
 
-import { ConnectionState, RxWebSocket } from './RxWebSocket';
+import { ConnectionState, WalletLinkWebSocket } from './RxWebSocket';
```

Comment:
Should we rename this file name to WalletLinkWebSocket?

Reply:
yes indeed. 
didn't make the change yet due to github diff issues. will address in a later pr.